specs/cli/cli.md

```diff
@@ -0,0 +1,55 @@
+# FAST CLI (WIP)
+
+## Overview
+FAST CLI is a set of tools used for creating, building and developing FAST projects.
+
+FAST CLI will have an independent code base and will be packaged and installable on any operating system.
+
+## Use Cases
+- A user can use FAST CLI to generate a new FAST design system project with a single command.
+- A user can generate a new component file structure in FAST using a single command in FAST CLI
+
+## Features
+- Project generator based on given name string
+- Component generator
+- Cross-platform
+- Context awareness. Detects whether the current directory contains the appropriate FAST dependency.
+
+## Prior Art/Examples
+- [Aurelia CLI](https://aurelia.io/docs/cli/basics/)
+- [Stencil CLI](https://stenciljs.com/docs/cli)
+- [Open Web Components](https://open-wc.org/docs/development/generator/)
+- [Angular CLI](https://cli.angular.io/)
+
+## API
+
+### Commands
+
+**Usage**:
+```
+$ fast [command] [arg-2] [--flag-1] [--flag-2]
```

Comment:
When using long arguments `--flag`, it's also common to include the short equivalent `-f`

Reply:
I think this should be handled on a case by case basis. I've notice a lot CLI tools going away from over-aliasing and reserving them only for frequently used flags. I can still update the spec to reflect an alias option, though.